src/Modules/SimplCommerce.Module.Catalog/Areas/Catalog/ViewModels/ProductThumbnail.cs

```diff
@@ -37,11 +49,13 @@ public class ProductThumbnail
 
         public double? RatingAverage { get; set; }
 
+        public long? TaxClassId { get; set; }
+
         public CalculatedProductPrice CalculatedProductPrice { get; set; }
 
-        public static ProductThumbnail FromProduct(Product product)
+        public static ProductThumbnail FromProduct(Product product, ICurrencyService currencyService)
```

Comment:
I dislike this change, but still thinking if we can do it in a better way

Reply:
Copy